modules/support_ticket/config/install/views.view.users_support_tickets.yml

```diff
@@ -340,13 +335,10 @@ display:
           delta_first_last: false
           multi_type: separator
           separator: ', '
-          field_api_classes: 0
-          date_format: 'time ago'
-          custom_date_format: ''
-          timezone: ''
+          field_api_classes: false
           entity_type: support_ticket
           entity_field: changed
-          plugin_id: date
+          plugin_id: field
```

Comment:
views is wanting to add this when reexporting:
-          settings: {  }
+          settings:
+            granularity: 2

but other than that it's just wanting to readd the uuid.